### ISSUE TITLE
ENH: Add helper to authenticate and save credentials to a JSON file path

### ISSUE THREAD

**Issue by tswast** [pydata/pydata-google-auth#22]
```
pydata_google_auth.save_json('path/to/credentials.json', use_local_webserver=True, client_id=OPTIONAL, client_secret=OPTIONAL)
```

This initiates the user authentication flow and saves the credentials to the path specified.

**Why?**

This will enable the use of application default credentials. Write your credentials to `/home/username/keys/google.json` and then run

```
export GOOGLE_APPLICATION_CREDENTIALS=/home/username/keys/google.json
```

from your shell. This bypasses the steps to create a service account and download a key file.

**Phase 2**

Expose this function as a command-line script. That way it can more easily be used for setting up application default credentials.

**Comment by tswast** [comment]
I just tried this today. This requires adding

```
  "type": "authorized_user",
```

to the JSON files we write.

**Comment by tswast** [comment]
Fixed by #24 and #25
